content/en/monitors/monitor_types/log.md

```diff
@@ -10,88 +10,84 @@ further_reading:
   text: "Schedule a downtime to mute a monitor"
 - link: "monitors/monitor_status"
   tag: "Documentation"
-  text: "Consult your monitor status"
+  text: "Check your monitor status"
 ---
 
-{{< img src="monitors/monitor_types/log/log_monitor_overview.png" alt="Log monitor overview" responsive="true" >}}
-
 ## Overview
 
-Log monitors alert when a specified type of log exceeds a user-defined threshold over a given period of time.
+A logs monitor alerts when a specified type of log exceeds a user-defined threshold over a given period of time.
+
+## Monitor creation
 
-## Setup
+To create a [logs monitor][1] in Datadog, use the main navigation: *Monitors --> New Monitor --> Logs*.
 
-Construct a query to control what is monitored:
+### Define the search query
 
-1. Define the search query:
-    {{< img src="monitors/monitor_types/log/define_the_search_query.png" alt="Define the search query" responsive="true" style="width:50%;" >}}
-    The search query has the same behavior as [the log explorer search][1]
-    
-2. Choose a [Measure][1] or [Facet][2] to monitor. [Measure][1] lets you choose the aggregation function whereas [Facet][2] displays a count.
-    
-    {{< img src="monitors/monitor_types/log/choose_measure_facet.png" alt="choose measure facet" responsive="true" style="width:50%;">}}
-    
-3. Select the aggregation function for the [Measure][1] you want to monitor:
-    
-    {{< img src="monitors/monitor_types/log/agg_function.png" alt="aggregation function for Log Analytics" responsive="true" style="width:50%;">}}
+As you define the search query, the graph above the search fields updates.
 
-4. (Optional) Define the alert grouping:
-  {{< img src="monitors/monitor_types/log/log_monitor_group_by.png" alt="Set alert conditions" responsive="true" style="width:50%;" >}}
-    With or without alert grouping defined, you get **one** alert when the aggregated value meets the conditions set below. Even if you split the query by host, a single notification is sent if several hosts meet the conditions set below. This is done to reduce notification noise.
+* If you have [multiple log indexes][2], select the index to search.
+* Construct a search query using the same logic as a [log explorer search][3].
+* Choose to monitor over a log count, [facet][4], or [measure][5]:
+    * **Monitor over a log count**: Use the search bar (optional) and do **not** select a facet or measure. Datadog evaluates the amount of logs over a selected time frame then compares it to the threshold conditions.
+    * **Monitor over a facet**: If a facet is selected, the monitor alerts over the `Unique value count` of the facet.
```

Comment:
We could clarify this because I believe this might create confusion and this could be due to the lack of screenshots (which I understand is intended).

Here are the things I have in mind:

1. User might not know what a facet is (we should likely add a link)
2. `If a facet is selected`, where? As you can select it in the search bar, in the "first" dropdown or in the group by one.

WDYT?

Reply:
This is written as a logical 1, 2, 3, 4 step process. It could be confusing because the UI isn't labeled. I added one screenshot to point this out. Also, related log terms are linked to the logs documentation.

Reply:
Awesome thanks a lot.